app/decorators/module_overview_decorator.rb

```diff
@@ -13,12 +13,13 @@ def call_to_action
     end
   end
 
-  # @return [Array<Hash{Symbol => String, Symbol, Array}>]
+  # @return [Hash{Symbol => String, Symbol, Array}]
   def sections
     mod.items_by_submodule.each.with_index(1).map do |(num, items), position|
       {
         heading: num.nil? ? 'Module introduction' : items.first.model.heading,
         position: position,
+        line_class: position == mod.items_by_submodule.size ? '' : 'line',
```

Comment:
My mentioning `Boolean` earlier imagined that it would be `line: true` to power the `unless` in the template which I think looks simpler. What do you think?